docs/installation.md

```diff
@@ -170,6 +177,8 @@ If you have ran 'react-native link` by mistake:
 6. delete ios/build folder
 7. start again with the installation steps
 
+If you use Xcode with version less than 9 you may get `use of undeclared identifier 'MKMapTypeMutedStandard'` error. In this case you have to update your Xcode.
```

Comment:
Here also `Entry, ":CFBundleIdentifier", Does Not Exist` may happen if not updated to XCode 9. 🙏

Reply:
@alvelig  grrr, merged while you were commenting :(

Reply:
@satori-ytolstoguzov if you want to update this I'll merge it, sorry for the mess 🍺